pinot-core/src/test/java/org/apache/pinot/core/common/DataFetcherTest.java

```diff
@@ -249,9 +249,7 @@ public void testFetchStringValues() {
   @Test
   public void testFetchBytesValues() {
     testFetchBytesValues(BYTES_COLUMN);
-    testFetchBytesValues(HEX_STRING_COLUMN);
     testFetchBytesValues(NO_DICT_BYTES_COLUMN);
```

Comment:
We can test on `STRING_COLUMN` and `NO_DICT_STRING_COLUMN`

Reply:
Done